inductiva/fluids/post_processing/openfoam.py

```diff
@@ -1,55 +1,55 @@
-"""Visualization processing of WindTunnel scenario.
+"""Post-processing tools of fluid dynamics steady-state simulations.
 
-This class implements various visualization capabilities for
-the WindTunnel scenario. Namely:
+This class implements various post-processing capabilities for
+the visuals associated. Namely:
     - Pressure over object;
     - Cutting plane;
-    - StreamLines.
+    - Stream lines.
 
-Currently, we only support the OpenFOAM simulator.
+Current Support - OpenFOAM
 """
 import os
 from dataclasses import dataclass
 from enum import Enum
 from typing import Literal
-import csv
 import pathlib
 
 import pyvista as pv
 
-from inductiva.types import Path
-from inductiva.utils.visualization import MeshData
-from inductiva.utils import files
+from inductiva import types, utils
 
 
-class WindTunnelOutput:
-    """Post-Process WindTunnel simulation outputs.
+class SteadyStateOutput:
+    """Post-Process steady-state simulation outputs.
 
     This class contains several methods to post-process the output 
-    and visualize the results of a WindTunnel simulation.
+    and visualize the results of a steady-state simulations where
+    time-independent results are obtained.
 
-    Current Support:
-        OpenFOAM
+    To be general we assume that a simulation is performed inside a
+    regular box and a certain object is placed inside the box.
+    The object is either a small object inside the domain or a more
+    general object that spreads all through the domain.
     """
 
-    def __init__(self, sim_output_path: Path):
-        """Initializes a `WindTunnelSimulationOutput` object.
+    def __init__(self, sim_output_path: types.Path):
+        """Initializes a `SteadyStateOutput` object.
 
         Args:
             sim_output_path: Path to simulation output files.
+        
+        Attributes:
+            sim_output_path: Path to simulation output files.
+            last_time_step: Last time step of the simulation.
+                Obtained through the output folders.
```

Comment:
I think for now it is not necessary, since for most the only attribute is equal to the arg - sim_output_path. I added here to refer to a new attribute computed it from that.